### DISCUSSION TITLE
Task05 Problem 4c

### DISCUSSION THREAD

**Discussion by NicolasFranklin** [chulminy/CIVE497-CIVE700#127] [General]
Is it expected to complete this portion of the assignment by hand? Questions 4a and 4b I am assuming need to be done by hand, but could 4c be done via Matlab as it is a bit tedious and repetitive? 

![image](https://user-images.githubusercontent.com/40611754/109394023-d8398880-78f2-11eb-85b4-e31728a4fccb.png)

**Comment by chulminy** [comment]
I have added this in the question:

(a) and (b) by hand
(c) using MATLAB

**Reply by rockyx50** [reply]
I already submitted the assignment a few days ago and did 4b by Matlab. Do I need to redo it and resubmit it?

**Reply by chulminy** [reply]
> I already submitted the assignment a few days ago and did 4b by Matlab. Do I need to redo it and resubmit it?

Can you do that? Sorry for the confusion.
